### PR TITLE
Added link to directions on how to share a scheme

### PR DESCRIPTION
**Pull request by jeeftor**
I added a link to Apple's documentation that shows how to enable Scheme sharing - as the documentation didn't really offer any good advice.

### CODE REVIEW

**Comment by robrix** [comment]
Great idea!

File: README.md [hunk 1]
```diff
@@ -116,7 +116,8 @@ The specific requirements of any framework project are listed below.
 
 Carthage will only build Xcode schemes that are shared from your `.xcodeproj`. You can see if all of your intended schemes build successfully by running `carthage build --no-skip-current`, then checking the [Carthage/Build][] folder.
 
-If an important scheme is not built when you run that command, open Xcode and make sure that the scheme is marked as “Shared,” so Carthage can discover it.
+If an important scheme is not built when you run that command, open Xcode and make sure that the scheme is marked as “Shared,” so Carthage can discover it.  Click [here](https://developer.apple.com/library/ios/recipes/xcode_help-scheme_editor/Articles/SchemeShare.html) for details.
```
Comment (robrix) [inline README.md]: Instead of the “click here” sentence, could you link the ‘scheme is marked as “Shared”’ bit please?

Comment (jeeftor) [inline README.md]: Will do!

**Comment by robrix** [comment]
:ok_hand: Perfect!

**Pull request merged**